### DISCUSSION TITLE
AMP

### DISCUSSION THREAD

**Discussion by therentabrain** [Eloston/ungoogled-chromium#2723] [General]
Surprised I didn't find any discussion about this already; maybe I just don't know where to look.

AMP links are originally designed for mobile devices, but can be shared, emailed, posted on reddit, and then clicked on by desktop.  Additionally, I am seeing indication that AMP is now expanding into the desktop world intentionally.

FTR, it's not the technology itself that bothers me at all (I am actually very much in favor of simplifying and stripping down the overbloated web page) but the fact that some amount (all?) of amp traffic runs through google's servers and is reformatted and tracked at their discretion.  

Does ungoogled chromium already avoid google's AMP gatekeeping natively?  

If not, is there a (trusted) plugin or setting for it? 

This looks good but it hasn't been updated in a while, and I see it may have incompatibilities at this point, e.g. with google image search:
https://github.com/da2x/amp2html

(I recognize that using google image search is choosing to be subject to google's monetizing of my traffic, but that's a choice I and my users want to make, not have it happen when visiting whatever arbitrary other independent page.)

Background information, if you (like me) weren't clear until today on what AMP is:

https://brave.com/privacy-updates/18-de-amp/ (note: I really dislike Brave but this is a useful description)
https://explainplease.co.uk/the-controversy-surrounding-amp-links-from-google/
https://www.reddit.com/r/AmputatorBot/comments/ehrq3z/why_did_i_build_amputatorbot/

**Comment by PF4Public** [comment]
> Does ungoogled chromium already avoid google's AMP gatekeeping natively?

No. If Google starts adding "AMP"s on the desktop, there might be incentive to investigate this matter further.
